tests/Collection/CrudSpecFunctionalTest.php

```diff
@@ -81,7 +86,7 @@ public function provideSpecificationTests()
      */
     private function assertEquivalentCollections($expectedCollection, $actualCollection)
     {
-        $mi = new MultipleIterator;
+        $mi = new MultipleIterator(MultipleIterator::MIT_NEED_ANY);
```

Comment:
I didn't realize when originally writing the test that [MultipleIterator](http://php.net/MultipleIterator)'s default behavior is to stop when any iterator stops yielding results. This caused us to not actually compare the actual and expected output collections because the actual collection had no documents (due to the pipeline never executing).

Note: the GridFS spec tests also use MultipleIterator for a similar assertion and would benefit from using `MIT_NEED_ANY`, but I'll take care of that in a separate PR.